src/components/EntitiesCount/EntitiesCount.tsx

```diff
@@ -0,0 +1,37 @@
+import {Label} from '@gravity-ui/uikit';
+
+interface EntitiesCountProps {
+    countTotal: number | string;
+    countCurrent?: number | string;
+    entityName?: string;
+    isLoadingStatus?: boolean;
+    className?: string;
```

Comment:
verbose naming
```
total
current
name/title/label
loading
```

loading instead of isLoading because it's a prop, not a variable
```
<EntitiesCount loading />
```
vs
```
<EntitiesCount isLoading />
```

Reply:
although i don't really like loading state in this component. This is a counter, it shouldn't know about loading state, it doesn't load anything
I understand why, but I suggest look for another way